workers/generic-worker/mounts.go

```diff
@@ -714,24 +714,37 @@ func ensureCached(fsContent FSContent, taskMount *TaskMount) (file string, err e
 	return
 }
 
-func extract(fsContent FSContent, format string, dir string, taskMount *TaskMount) error {
+func extract(fsContent FSContent, format string, dir string, taskMount *TaskMount) (err error) {
 	cacheFile, err := ensureCached(fsContent, taskMount)
```

Comment:
I think this `err` will shadow the one in the function signature, so to be safe I think you will need to change this to:
```go
string cacheFile
cacheFile, err = ensureCached(fsContent, taskMount)
```

Reply:
Ahh crap, good catch thank you